src/tests/ftest/io/romio.py

```diff
@@ -31,8 +31,6 @@
 
 sys.path.append('./util')
 sys.path.append('../util')
-sys.path.append('../../../utils/py')
```

Comment:
Since changes are being made here, shouldn't lines 32-33 also be removed (and most likely line 27 assuming `sys` is not used elsewhere)?

Reply:
I found this change by grepping the src for "utils/py" which is a directory that used to exist but doesn't any more, so I know L34 was unneeded.  For L32-33 the directories I did check the directories exists at least, I don't know if they actually need to be added to PATH or not.

If sys is not used then once pylint checking is working then I'd expect that to throw an error, and we can fix it at that point.